frontend/packages/kubevirt-plugin/src/components/modals/menu-actions-modals/utils.ts

```diff
@@ -10,3 +11,12 @@ export const redirectToList = (vmi: VMGenericLikeEntityKind, tab?: 'templates' |
     history.push(`/k8s/ns/${getNamespace(vmi)}/${VIRTUALMACHINES_BASE_URL}/${tab || ''}`);
   }
 };
+
+export const redirectSapHanaTemplate = (template: TemplateKind, preselectedNamespace?: string) => {
```

Comment:
The original idea was to render the template instantiate page, not redirect to it.

This can work here because it's just a quick fix for SAP HANA, but i'm curious, any reason we redirect to to a new page and not rendering it ?